en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/performance-tests/router-with-one-cpu-for-rest-api.md

```diff
@@ -0,0 +1,74 @@
+These test results were obtained considering a Choreo Connect deployment with default values. Therefore the concurrency level used for the tests was two. Below table shows resource allocations for the Choreo Connect components.
```

Comment:
Let's add a proper title for this page.

Reply:
fixed with: https://github.com/wso2/docs-apim/pull/6099/commits/f93dd8263b73034b73bc8e85c773c059c0bed41e